### PR TITLE
Add documentation for notebooks image update process

### PR DESCRIPTION
**Pull request by atheo89**
Resolves: https://github.com/opendatahub-io/notebooks/issues/59 

## Description
Added documentation for notebooks image update process 

- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: UPDATES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Naming Convention for Notebook Images
+
+To ensure consistency in naming of the notebook images, we are using the convention 'workbench-images:image-name-tag'. Where, the "image-name" segment should indicate the image's type/flavor, along with its OS version and Python version (e.g., jupyter-datascience-ubi8-python-3.8), and the "tag" segment should include the release name and patch version for easy identification. For clarity, we suggest using a YearIndex scheme for the release (e.g., 2023a) and incorporating the patch's date and weekly tag in the YYYYMMDD format.
+
+For instance, "*workbench-images:jupyter-datascience-ubi9-python-3.9-2023a-2023011*5" refers to the Jupyter Standard Data Science image based on UBI9, using Python 3.9, with a set of components as defined in release 2023a. This image is fully patched as of January 15, 2023. Additionally, "*workbench-images:jupyter-datascience-ubi9-python-3.9-2023a-weekly*" denotes the most recent version of the image.
```
Comment (harshad16) [inline UPDATES.md:20]:
```suggestion
For instance, "*workbench-images:jupyter-datascience-ubi9-python-3.9-2023a-20230115*" refers to the Jupyter Standard Data Science image based on UBI9, using Python 3.9, with a set of components as defined in release 2023a. This image is fully patched as of January 15, 2023. Additionally, "*workbench-images:jupyter-datascience-ubi9-python-3.9-2023a-weekly*" denotes the most recent version of the image.
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just one formatting change needed

File: UPDATES.md (hunk 1 above)
Comment (atheo89) [reply UPDATES.md]: Thanks for the observation! I added the change

**Review by harshad16 — COMMENTED** [review]
/lgtm
:+1:

**Pull request merged**